datadog/provider.go

```diff
@@ -228,5 +228,10 @@ func translateClientError(err error, msg string) error {
 }
 
 func getUserAgent(clientUserAgent string) string {
-	return fmt.Sprintf("terraform-provider-datadog/%s (terraform %s) %s", version.ProviderVersion, meta.SDKVersionString(), clientUserAgent)
+	return fmt.Sprintf("terraform-provider-datadog/%s (go %s; terraform %s; terraform-cli %s) %s",
+		version.ProviderVersion,
+		runtime.Version(),
```

Comment:
This one may appear twice (as I believe its defined inside the datadogV1 user agent header, but I don't think it'll be an issue